transformer_engine/pytorch/csrc/extensions/quantizer.cpp

```diff
@@ -109,6 +109,7 @@ std::pair<TensorWrapper, py::object> Float8Quantizer::create_tensor(
   }
   const py::object py_columnwise_data = create_transpose ? py::cast(columnwise_data) : py::none();
   opts = opts.dtype(torch::kFloat32);
+  // TODO: Replace with an empty tensor.
```

Comment:
Why can't we do that now?

Reply:
I tried to do it but it seems the userbuffer and fp8 dpa rely on it. We probably need a dedicated PR to remove it.